src/transform.ts

```diff
@@ -46,6 +46,22 @@ export function addCoersion(def: any): void {
   }
 }
 
+export function findRefs(definition: any): string[] {
```

Comment:
This seems to be the application of some function `ref` to a higher-order function, e.g. `deepFlatMap`.  I think writing it that way may make it easier to understand.   You might even find a suitable HOF in Ramda.

Reply:
Probably need a `deepFind`, I'll write one.